enterprise/api/v1/views.py

```diff
@@ -167,6 +169,28 @@ def courses(self, request, pk=None):  # pylint: disable=invalid-name,unused-argu
         serializer.update_enterprise_courses(enterprise_customer, catalog_id=enterprise_customer.catalog)
         return get_paginated_response(serializer.data, request)
 
+    @detail_route(methods=['post'])
+    def course_enrollments(self, request, pk):  # pylint: disable=invalid-name,unused-argument
+        """
+        Creates a course enrollment for an EnterpriseCustomerUser.
+        """
+        enterprise_customer = self.get_object()
+        serializer = serializers.EnterpriseCustomerCourseEnrollmentsSerializer(
+            data=request.data,
+            context={
+                'enterprise_customer': enterprise_customer,
+                'request_user': request.user,
+            }
+        )
```

Comment:
It would be nice if this endpoint could take a list of course enrollments to create/update instead of just one. It [looks like](http://www.django-rest-framework.org/api-guide/serializers/#dealing-with-multiple-objects) you can do this by passing a `many=True` keyword argument to the `EnterpriseCustomerCourseEnrollmentsSerializer` constructor.

Reply:
I'll look into it.

Reply:
Unfortunately this is not as simple as adding that param. I've been trying to get it right for the entire morning but want to ask how important it is to support bulk before I go further into the rabbit hole :-)

Reply:
Maybe we can take a look at it together tomorrow morning and you can show me what is going on.